mysql/resource_ti_resource_group_user_assignment.go

```diff
@@ -88,8 +93,15 @@ func ReadResourceGroupUser(ctx context.Context, d *schema.ResourceData, meta int
 		return diag.Errorf(`error getting user %s`, err)
 	}
 
+	// If the user doesn't exist, instead of erroring, recognize that there's
+	// terraform drift and attempt to create the assignment again.
+	if user == "" {
+		d.SetId("")
+		return nil
+	}
+
 	d.Set("user", user)
-	d.Set("resourceGroup", resourceGroup)
+	d.Set("resource_group", resourceGroup)
```

Comment:
This was resulting in weird tf diffs - fixing so that we see the correct rg state!